import/municipalities.yaml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
I don't know why we need `---` here since we have only one document in this file. But I did notice that's how the other imports were set up. So :+1: for following the pre-existing convention.

Reply:
Yes, there was no need to add this.
Yes only reason to add this is follow the previous convention.